glances/exports/mqtt/__init__.py

```diff
@@ -38,13 +38,17 @@ def __init__(self, config=None, args=None):
 
         # Load the MQTT configuration file
         self.export_enable = self.load_conf(
-            'mqtt', mandatories=['host', 'password'], options=['port', 'user', 'topic', 'tls', 'topic_structure']
+            'mqtt', mandatories=['host', 'password'], options=['port', 'user', 'hostname', 'topic', 'tls', 'topic_structure']
```

Comment:
Variable name is confusing with host

Reply:
okay, what do you think of 'devicename'?
I chose hostname because this variable is already used in __init__.py